### PR TITLE
Move all command definitions out of main.go into implementation source

### PR DESCRIPTION
**Pull request by rhatdan**
This will allow us to more easily share these source modules between
kpod and buildah as well as move keep the definition with the implementation.

### CODE REVIEW

File: cmd/buildah/mount.go [hunk 1]
```diff
@@ -8,6 +8,13 @@ import (
 
 var (
 	mountDescription = "Mounts a working container's root filesystem for manipulation"
+	mountCommand     = cli.Command{
+		Name:        "mount",
+		Usage:       "Mount a working container's filesystem root",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/mount.go]: should this be "root filesystem" like line 10?

File: cmd/buildah/umount.go [hunk 2]
```diff
@@ -8,6 +8,14 @@ import (
 
 var (
 	umountDescription = "Unmounts a working container's root filesystem"
+	umountCommand     = cli.Command{
+		Name:        "umount",
+		Aliases:     []string{"unmount"},
+		Usage:       "Unmount a working container's filesystem root",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/umount.go]: should this be root filesystem like line 10 too?

File: cmd/buildah/main.go [hunk 3]
```diff
@@ -69,82 +69,16 @@ func main() {
...
-			ArgsUsage:   "CONTAINER-NAME-OR-ID IMAGE",
-		},
-		{
-			Name:        "delete",
-			Usage:       "Delete working container(s)",
-			Description: deleteDescription,
-			Action:      deleteCmd,
-			ArgsUsage:   "CONTAINER-NAME-OR-ID [...]",
-		},
+		addCommand,
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/main.go]: Do you need an import for these commands as they're in other source files?  I'm not seeing at least a '/cmd' in the imports and thought there'd be one (but may be showing my lack of go xp).

Comment (rhatdan) [inline cmd/buildah/main.go]: As long as they are in the same package, these fields are global.

Comment (TomSweeneyRedHat) [inline cmd/buildah/main.go]: kk, thx for the 411.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ ad2d49b

**Pull request closed without merging**